hub/powertoys/paste-as-plain-text.md

```diff
@@ -0,0 +1,32 @@
+---
+title: PowerToys Paste as Plain Text for Windows
+description: Allows you to paste text from your clipboard, excluding text-formatting, with a quick key shortcut.
+ms.date: 02/10/2023
+ms.topic: article
+ms.localizationpriority: medium
+no-loc: [PowerToys, Windows, PowerToys Paste as Plain Text, Paste as Plain Text, Win]
+---
+
+# Paste as Plain Text
+
+Allows you to paste text from your clipboard, excluding text-formatting, with a quick key shortcut.
```

Comment:
I feel we should directly comment it will replace your clipboard with the unformatted version.

Reply:
```suggestion
PowerToys **Paste as plain text** enables you to paste text stored in your clipboard, excluding any text-formatting, using a quick key shortcut. Any formatting included with the clipboard text will be replaced with an unformatted version of the text.
```